metron-platform/metron-solr/metron-solr-common/src/main/java/org/apache/metron/solr/dao/SolrSearchDao.java

```diff
@@ -91,29 +106,28 @@ public SearchResponse search(SearchRequest searchRequest, String fieldList)
       throw new InvalidSearchException(
           "Search result size must be less than " + accessConfig.getMaxSearchResults());
     }
-    try {
-      SolrQuery query = buildSearchRequest(searchRequest, fieldList);
-      QueryResponse response = client.query(query);
-      return buildSearchResponse(searchRequest, response);
-    } catch (SolrException | IOException | SolrServerException e) {
-      String msg = e.getMessage();
-      LOG.error(msg, e);
-      throw new InvalidSearchException(msg, e);
+  }
+
```

Comment:
The only thing that comes to mind with this, is if we have multiple people/ui/api's  doing queries at the same time when we are in a 'state' or something, it is going to be a mess.  I don't know what context you can add though.

Reply:
Hm, that's true. The main target for this change is folks doing performance tuning, but I think you're right. If you turn this on to see prod runtime issues, having many clients is going to show these logs in a mixed order. How about a random hash to prepend the lines with a random ID? At least the info can be grouped then. Or I could build this into one big massive string so there's only one log entry?

Reply:
We can add that as a follow on really, I just wanted to point that out.   We should be explicit about the intended use case here.